engine/src/main/resources/application.conf

```diff
@@ -36,6 +36,9 @@ system {
 
   // If 'true', a SIGINT will trigger Cromwell to attempt to abort all currently running jobs before exiting
   abort-jobs-on-terminate = false
+
+  // Max number of retries per job that the engine will attempt in case of a retryable failure received from the backend
+  maxRetries = 3
```

Comment:
@mcovarr Are you talking just about the property, or this feature as a whole? As in, do we want the engine to not have a limit and *always* retry a job if we receive a retry able failure? (relying on the backend to have a limit on it's side)